server/config-docs.ts

```
@@ -150,18 +150,60 @@ const validator = ajv.compile({
  * Also we check that all links ends with "/: for consistency.
  */
 
-const normalizeDocsUrl = (version: string, url: string, raw?: boolean) => {
+/*
+ * normalizeDocsUrl ensures that internal docs URLs include trailing slashes and
+ * adds the docs version to the URL. It also throws an error if the docs page
+ * corresponding to the URL does not exist. The assumption is that docs pages
+ * are stored in content/<version>/docs/pages. Callers can ignore errors for
+ * nonexistent files by setting `checkExistence` to false.
+ *
+ */
+export const normalizeDocsUrl = (
+  version: string,
+  url: string,
+  checkExistence: boolean = true
```

Can we add a test to make sure `checkExistence` correctly ignores errors